mandatory/1-fix-functions.js

```diff
@@ -10,18 +10,16 @@
 
 */
 
-function getMood() {
-  let isHappy = true;
-
-  if (isHappy) {
+function getMood(isHappy) {
+ if (isHappy === true) {
```

Comment:
I have seen others write a true/false condition in a shorthand way as follows:
instead of writing
if (isHappy === true) {
instead, you could write
if (isHappy) {

I believe both those mean the same thing.

Reply:
thank you@malkit